### PR TITLE
Fix AVA-related CI testing issues

### PR DESCRIPTION
**Pull request by fregante**
Travis started failing at some point, probably because tests create thousands of parallel threads and at some point we reached the limit.

https://travis-ci.org/sindresorhus/refined-github/builds/647752010

### CODE REVIEW

**Comment by fregante** [comment]
Now AVA seems to be failing 🤷‍♂️ 
cc @sindresorhus

**Comment by sindresorhus** [comment]
Maybe because of https://github.com/avajs/ava/releases/tag/v3.3.0 ?

**Comment by fregante** [comment]
But we have a lockfile

https://github.com/sindresorhus/refined-github/blob/a55748c84ad87c56a813dbd0e281665c1df0a7cf/package-lock.json#L2040-L2042

**Comment by sindresorhus** [comment]
Right, forgot we have that. Must be either that Travis changed something then, or more likely, the Node.js version got a patch/minor release which changed something.

**Comment by fregante** [comment]
Possible. I remember seeing Node 10 on GH Actions but now I see v12.13.1

https://help.github.com/en/actions/reference/software-installed-on-github-hosted-runners#ubuntu-1804-lts

**Comment by fregante** [comment]
Post-mortem: AVA testing needs to be upgraded so that Node 13 is supported.
And I don't know what's up with Travis, but serial tests and Node 11 work.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -14,8 +14,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
```
Comment (sindresorhus) [inline .github/workflows/test.yml]: You should also upgrade to `checkout@v2` ;)

https://github.com/actions/checkout/releases/tag/v2.0.0

Comment (fregante) [reply .github/workflows/test.yml]: pr welcome

**Pull request merged**